src/Proxy/SpyProxy.php

```
@@ -23,19 +24,32 @@
 
 namespace Bottledcode\DurablePhp\Proxy;
 
+use ReflectionClass;
+use ReflectionMethod;
+use ReflectionParameter;
+
 class SpyProxy extends Generator
 {
-    protected function pureMethod(\ReflectionMethod $method): string
+    protected function pureMethod(ReflectionMethod $method, bool $isHook = false): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$isHook'. (Unused Code Rules)

(UnusedFormalParameter)